packages/samples/angular/tsconfig.app.json

```diff
@@ -4,7 +4,10 @@
   "compilerOptions": {
     "outDir": "./out-tsc/app",
     "types": [],
-    "paths": {"@angular/*": ["../../../node_modules/@angular/*"]},
+    "paths": {
+      "@angular/*": ["../../../node_modules/@angular/*"],
+      "@coveo/atomic-angular": ["../../atomic-angular/projects/atomic-angular/dist/"]
```

Comment:
It looks like even with the script to link dependencies, Angular is failing to run in dev. This script fixes it. This script alone isn't enough either, since other dependencies fail and adding them here doesn't fix the issue. Everything seems very inconsistent and I couldn't find the exact reason.